src/main/java/marquez/dao/RunArgsDAO.java

```diff
@@ -0,0 +1,21 @@
+package marquez.dao;
+
+import marquez.core.models.RunArgs;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindBean;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+@RegisterRowMapper(RunArgsRow.class)
+public interface RunArgsDAO extends SqlObject {
```

Comment:
A reminder that we don't need to extend `SqlObject` if we don't need direct access to the handler.

Reply:
Ah yeah, this was copy pasta. Looking at the JDBI doc, this is just an optional mixin, so 👍

Reply:
Looks like we used to need it when we were using `getHandle` to do things in transactions, but now that we're standardizing on the `@Transaction` annotation, we can remove this, which is good!